packages/outline-core-link/src/outline-core-link.ts

```diff
@@ -0,0 +1,104 @@
+import { html, TemplateResult, CSSResultGroup } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+// Our base component, which all others extend.
+import { OutlineElement } from '@phase2/outline-core';
+
+import type { LinkTargetType, LinkRelType } from './config';
+import componentStyles from './outline-core-link.css.lit';
+
+/**
+ * The Outline Core Link component
+ *
+ * @element outline-core-link
+ * @extends OutlineElement
+ * @slot - The default slot for this element.
+ * @cssprop --outline-link-transition-property: The CSS transition property to use for the link.
+ * @cssprop --outline-link-transition-duration: The CSS transition duration to use for the link.
+ * @cssprop --outline-link-transition-timing-function: The CSS transition timing function to use for the link.
+ * @cssprop --outline-link-color-default: The default link color.
+ * @cssprop --outline-link-color-default-decoration: Text decoration for the default link.
+ * @cssprop --outline-link-color-hover: The link color when hovering on the link.
+ * @cssprop --outline-link-color-hover-decoration: Text decoration for the link when hovering.
+ * @cssprop --outline-link-color-focus: The link color when focusing on the link.
```

Comment:
This docblock should replace `outline-link` with `outline-core-link` in all instances, as well as likely check that all variables are represented here.

Reply:
What should we do with extending components?
Should Figma tokens use outline-core-link or outline-link (extended at the project level) variables?